replication/replication.proto

```diff
@@ -213,3 +231,27 @@ message GetVolumeReplicationInfoResponse {
   // This field is REQUIRED.
   .google.protobuf.Timestamp last_sync_time = 1;
 }
+// Specifies what source the replication will be created from. One of the
+// type fields MUST be specified.
+message ReplicationSource {
```

Comment:
(question) As we intend to reuse this message for both a replication source and a destination, how easy is it to modify the spec to make this a (say)`ReplicationEntity` and in one of the calls above use it as `replication_source` and later a `replication_destination`?

Reply:
CSI spec [example](https://github.com/container-storage-interface/spec/blob/bd1cb6579c9ed791a6d9980fa3c0ef11600cc273/csi.proto#L350-L370)

Reply:
Frankly speaking, this example is the source of my inspiration. ReplicationSource is copied from it. So, you are suggesting only to change name from ReplicationSource to ReplicationEntity? Am I right?

Reply:
Discussed with @Madhu-1 we MAY have different source|destination IDs or fields, and it is difficult to change from a common message to a specific one, hence retaining it as is in this PR us useful.